src/util.cpp

```diff
@@ -953,3 +957,12 @@ fs::path AbsPathForConfigVal(const fs::path& path, bool net_specific)
 {
     return fs::absolute(path, GetDataDir(net_specific));
 }
+
+int ScheduleBatchPriority(void) {
+#if HAVE_SCHED_H && defined(SCHED_BATCH)
+    sched_param param{.sched_priority = 0};
+    return sched_setscheduler(0, SCHED_BATCH, &param);
```

Comment:
Would calling `pthread_setschedparam` instead of `sched_setscheduler` be a little more portable? It seems like you might not need the configure check using this.